tcms/testruns/tests/test_views.py

```diff
@@ -473,11 +474,15 @@ def test_redirect_to_run_if_still_case_runs_exist_after_removal(self):
                              reverse('testruns-get', args=[self.test_run.pk]))
 
     def test_redirect_to_add_case_runs_if_all_case_runs_are_removed(self):
+        case_run = []
+
+        for case_run in self.test_run.case_run.all():
+            case_run.add(case_run.pk)
```

Comment:
argh, both the loop variable and the list are named the same. Change one of the names and pay attention to whether we hold a single value or a collection of values. Names are important!